pandas/tests/util/test_assert_extension_array_equal.py

```diff
@@ -32,22 +32,21 @@ def test_assert_extension_array_equal_not_exact(kwargs):
         tm.assert_extension_array_equal(arr1, arr2, **kwargs)
 
 
-@pytest.mark.parametrize(
-    "check_less_precise", [True, False, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
-)
-def test_assert_extension_array_equal_less_precise(check_less_precise):
-    arr1 = SparseArray([0.5, 0.123456])
-    arr2 = SparseArray([0.5, 0.123457])
+@pytest.mark.parametrize("decimals", [0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+def test_assert_extension_array_equal_less_precise(decimals):
+    rtol = 1 * 10 ** -decimals
+    arr1 = SparseArray([0.5, 0.12345])
+    arr2 = SparseArray([0.5, 0.12346])
 
-    kwargs = dict(check_less_precise=check_less_precise)
+    kwargs = dict(rtol=rtol)
 
-    if check_less_precise is False or check_less_precise >= 5:
+    if decimals >= 5:
         msg = """\
 ExtensionArray are different
 
 ExtensionArray values are different \\(50\\.0 %\\)
-\\[left\\]:  \\[0\\.5, 0\\.123456\\]
-\\[right\\]: \\[0\\.5, 0\\.123457\\]"""
+\\[left\\]:  \\[0\\.5, 0\\.12345\\]
+\\[right\\]: \\[0\\.5, 0\\.12346\\]"""
```

Comment:
is the test currently wrong or is this changing the expected behavior?

Reply:
This was changed in one of those early diffs. I'm reverting it because now it's clearer how the old `check_less_precise` maps to the new `rtol`